### DISCUSSION TITLE
3ds max look dev publish

### DISCUSSION THREAD

**Discussion by ronnyascencio** [masqu3rad3/tik_manager4#137] [General]
Hello,  I have a doubt if i publish the archive   from a lookdev task this archive can be referenced in other scene ?  like normal xref   when i import  from tik in the new scene?
 i hope it makes sense my question/doubt

**Comment by masqu3rad3** [comment]
Hi @ronnyascencio,
In order to be able to extract (export) a format along with a publish it requires an extractor.
In order to be able to ingest (import or reference for example) you need an ingestor.

Currently there are basic ingestors and extractors for the available DCCs, however these can be easily expanded or modified for personal use or studio requirements.

Can you elaborate this a little bit more? Which DCC you are referring for? Essentially if it is something you can do it manually, Tik Manager can automate it.

**Reply by masqu3rad3** [reply]
Btw, I just finished Alembic and USD extractors and ingestors for 3dsMax and improved FBX ones. They will be available on next release

**Comment by ronnyascencio** [comment]
juts  to know how much can i stay in 3ds max  in all steps of the project,  from model to lighting  with out writing  alembics or usd or something like that  using xref or so   if that where the case, thanks for the answer!
